setup/security/authorization/pipeline-permissions/index.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: single
+title:  "Pipeline Permissions"
+sidebar:
+  nav: setup
+---
+
+{% include alpha version="1.9" %}
+
+{% include toc %}
+
+Pipeline permissions enable automatically triggered pipelines to modify
+resources in protected accounts and applications. They are an alternative
+to manually managing (Fiat Service Accounts)[../service-accounts/].
+
+Without pipeline permissions, a Spinnaker operator first has to first create a
+Fiat Service account with the correct permissions. A user can then specify the
+service account as the `RunAsUser` per automated trigger. Pipeline permissions
+simplifies this flow -- a user only has to specify the set of roles in the
+pipeline specification based on which a Fiat service account is automatically
+created and associated with the pipeline.
+
+## Enabling pipeline permissions
+
+Pipeline permissions are disabled by default and can be enabled by
+setting the following flags:
+
+* For Orca, add the following to  `orca-local.yml`
```

Comment:
I'm imaging that you're looking for community feedback before adding this to halyard?

Reply:
Yeah. Good idea. Created https://github.com/spinnaker/spinnaker/issues/3124